spec.html

```diff
@@ -34865,6 +35091,20 @@ <h1>Static Semantics: CharacterValue</h1>
           1. Let _ch_ be the code point matched by |IdentityEscape|.
           1. Return the code point value of _ch_.
         </emu-alg>
+        <emu-grammar>ClassCharacter :: [lookahead &notin; ClassReservedDouble] SourceCharacter but not ClassSyntaxCharacter</emu-grammar>
```

Comment:
```suggestion
        <emu-grammar>ClassCharacter :: SourceCharacter but not ClassSyntaxCharacter</emu-grammar>
```
We also don't usually put lookahead-constraints in non-defining productions.

(Similarly throughout.)